### PR TITLE
Supoprt multiple spawn

### PR DESCRIPTION
**Pull request by yuokamoto**

### CODE REVIEW

File: ExternalTest/rr_sim_tests/docker/Dockerfile [hunk 1]
```diff
@@ -33,5 +35,5 @@ RUN apt update && apt install -y python3-pip ros-foxy-tf-transformations && pip3
 RUN cp ${ROS2_WS}/src/rr_sim_tests/docker/fastdds_setup.sh /
 RUN cp ${ROS2_WS}/src/rr_sim_tests/docker/fastdds_config.xml /
 
-CMD /bin/bash -c ". ${ROS2_WS}/install/setup.bash && source fastdds_setup.sh && ros2 run rr_sim_tests test_random_spawn --ros-args -p robot_name:=${ROBOT_NAME}"
+CMD /bin/bash -c ". ${ROS2_WS}/install/setup.bash && source fastdds_setup.sh && ros2 run rr_sim_tests test_random_spawn --ros-args -p robot_name_prefix:=${ROBOT_NAME} -p start_index:=${START_INDEX} -p robot_num:=${ROBOT_NUM}"
```
Comment (larryng-rapyuta) [inline ExternalTest/rr_sim_tests/docker/Dockerfile]:
```suggestion
CMD /bin/bash -c ". ${ROS2_WS}/install/setup.bash && source fastdds_setup.sh && ros2 run rr_sim_tests test_random_spawn --ros-args -p robot_name_prefix:=${ROBOT_NAME_PREFIX} -p start_index:=${START_INDEX} -p robot_num:=${ROBOT_NUM}"
```
I think It supposed to be the prefix?

Comment (yuokamoto) [reply ExternalTest/rr_sim_tests/docker/Dockerfile]: ah yes. thank you.

**Review by larryng-rapyuta — APPROVED** [review]
tested and working

File: Source/RapyutaSimulationPlugins/Private/Robots/RRBaseRobot.cpp [hunk 2]
```diff
@@ -102,65 +102,60 @@ bool ARRBaseRobot::IsAuthorizedInThisClient()
...
+            res = false;
+        } else if (ROS2Interface->ROSSpawnParameters->GetNetworkPlayerId() ==
+                   npc->ROS2SimStateClient->GetNetworkPlayerId()) {
+            UE_LOG(LogRapyutaCore,
+                   Log,
+                   TEXT("[%s] [ARRBaseRobot::IsAuthorizedInThisClient()] PlayerId is matched. PlayerId=%d."),
+                   *GetName(),
+                   npc->GetPlayerState<APlayerState>()->GetPlayerId());
+            res = true;
+        } else {
```
Comment (Tadinu) [inline Source/RapyutaSimulationPlugins/Private/Robots/RRBaseRobot.cpp]: clang-format?

**Pull request merged**